recipes/openexr/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+from conans.model.conan_file import ConanFile
+from conans import CMake, tools
+import os
+
+
+class DefaultNameConan(ConanFile):
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
```

Comment:
Remove imports, `run_environment=True` should be enough for finding dynamic libraries.